apps/design-land/src/app/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,5 @@
+<daff-checkbox-set [formGroup]="checkboxes">
+  <daff-checkbox formControlName='option1' value='option1'>Option 1 </daff-checkbox>
```

Comment:
Update the single quotes to double quotes for consistent syntax in HTML files.

Reply:
Ill make a note for this in future PRs as well. fixed in a19666774303a5986f9a4617acdc5a58ec028acf